### PR TITLE
feat(docs): add customer tag docs

### PR DESCRIPTION
**Pull request by tobiasweiss93**

### CODE REVIEW

File: docs/meshstack.tag-schema.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      "maxLength": 4,
+      "pattern": "^\\d{4}$",
+      "restricted": true
+    }
+  },
+  "required": [
+    "costCenter"
+  ]
+}
+```
```
Comment (JohannesRudolph) [inline docs/meshstack.tag-schema.md:62]: d: we should discuss here that tags are available in LZs and that these may follow special rules / patterns. See also https://app.clickup.com/t/4gq3hu

File: docs/meshstack.tag-schema.md [hunk 2]
```diff
@@ -1,27 +1,107 @@
...
+## Use Cases for Metadata Tags
 
-Currently we support this functionality for meshCustomer and meshProjects.
+meshStack often acts as an organization's central registry of cloud environments and services. It therefore needs to cover
+a wide range of use cases for orchestrating organizational processes across multiple clouds, for example
 
-## Customer Tag Schema
+- centralized management and configuration of [cloud platforms](./meshcloud.platform-location.md)
+- life-cycle management for [cloud tenants](meshcloud.tenant.md) and [security policy automation](./meshcloud.landing-zones.md)
+- centralized Identity and Access management (IAM) via [multi-cloud identity federation](meshstack.identity-federation.md)
```
Comment (chewbeccy) [inline docs/meshstack.tag-schema.md:15]:
```suggestion
- centralized Identity and Access Management (IAM) via [multi-cloud identity federation](meshstack.identity-federation.md)
```

File: docs/meshstack.tag-schema.md [hunk 3]
```diff
@@ -1,27 +1,107 @@
...
+- unified [chargeback process](./meshcloud.project-metering.md) across all cloud platforms and services
 
-The Customer Tag Schema can define for example a costCenter and customerOwner. These tags can be edited as Administrator in the customers overiew. There exists for each customer the option to edit also the specifc tags.
+Many of these use-cases can require organizational metadata to be shared across different systems. Common examples of
```
Comment (chewbeccy) [inline docs/meshstack.tag-schema.md:18]:
```suggestion
Many of these use cases can require organizational metadata to be shared across different systems. Common examples of
```

File: docs/administration.customers.md [hunk 4]
```diff
@@ -23,10 +23,26 @@ With meshcloud you can easily create customer accounts.
...
+
+1. Login to the meshPanel with your credentials.
+2. Navigate to the **Administration** area ia the toolbox on the top right and then click on **Customers** in the menu.
+3. In the **Actions** column click on the **Invitation Link** button (it is only visible if it is enabled by the operator).
+4. Copy the link and give it to the people who should register a new customer.
+
+### Approve new Customer Accounts
+
+A meshStack installation can be configured to require a manual approval by a partner/admin for new meshCustomer registrations. Users who register receive an email notifying them, that their account is currently being verified. As soon as a partner/admin approves the request, they receive another email. This email informs the user, that he is now approved and can start working with the cloud ressources. Dependend on the configuration, the user may also have to confirm his email address via a confirmation link, that is available in the second email.
```
[the remark this answers is not in the document]
Comment (JohannesRudolph) [reply docs/administration.customers.md:39]: good catch - we should add a note to Contributing.md that gender specific language shall be avoided in the documentation

File: docs/administration.projects.md [hunk 5]
```diff
@@ -16,7 +28,13 @@ To edit the quotas, please follow these steps:
 5. Also under the **Actions** column, you reach the **Quota Dashboard**. These quotas are project-specific.
 6. You can now change the quotas as desired.
 
-## Project Export
+### Delete Projects
+
+When a user deletes a meshProject it is not completely removed in the cloud platforms and in the database, but first only flagged for deletion. An asynchronous background job removes tenants from platforms regularly. Partners and Admins can find a list of **Deleted Projects** in the **Projects** section of the **Administration** Area. This list allows Cloud Admins to check for projects that have been deleted. The "Deleted in platforms on" column shows when the background process that actually deletes the "marked for deletion" projects in the platforms has been executed successfully.
```
[the remark this answers is not in the document]
Comment (JohannesRudolph) [reply docs/administration.projects.md:33]: It's possible as long as the meshProject contains no meshTenants for which deletion is not (yet) possible

**Pull request merged**